### PR TITLE
fix external host setting for sqs

### PR DESCRIPTION
**Pull request by baermat**
Branch to make External Hostnames available for the new asf SQS provider available

### CODE REVIEW

File: tests/integration/test_sqs.py [hunk 1]
```diff
@@ -361,6 +361,33 @@ def test_batch_send_with_invalid_char_should_succeed(self, sqs_client, sqs_creat
...
+        SQS_PORT_EXTERNAL = "SQS_PORT_EXTERNAL"
+
+        monkeypatch.setattr(config, SQS_PORT_EXTERNAL, external_port)
+        monkeypatch.setattr(config, "HOSTNAME_EXTERNAL", external_host)
+        # TODO: remove once the old provider is discontinued
+        from localstack.services.sqs import sqs_listener as old_sqs_listener
+
+        monkeypatch.setattr(old_sqs_listener, SQS_PORT_EXTERNAL, external_port)
+
+        os.environ[SQS_PORT_EXTERNAL] = external_port
```
Comment (whummer) [inline tests/integration/test_sqs.py:379]: nitpick: Let's better use `monkeypatch.setenv(..)` here (to make sure the env gets reset after the test is finished)

File: tests/integration/test_sqs.py [hunk 2]
```diff
@@ -361,6 +361,33 @@ def test_batch_send_with_invalid_char_should_succeed(self, sqs_client, sqs_creat
         result_send = sqs_client.send_message_batch(QueueUrl=queue_url, Entries=batch)
         assert len(result_send["Failed"]) == 1
 
+    @pytest.mark.skipif(
```
Comment (whummer) [inline tests/integration/test_sqs.py:364]: nit: We could use the `@only_localstack` decorator here (from `fixtures.py`).

**Review by whummer — APPROVED** [review]
Looks great, thanks for the fix @baermat ! 👍 (only added two minor nitpicks..)

File: tests/integration/test_sqs.py (hunk 2 above)
Comment (baermat) [reply tests/integration/test_sqs.py:364]: yes we should definitely use that :)

File: tests/integration/test_sqs.py (hunk 1 above)
Comment (baermat) [reply tests/integration/test_sqs.py:379]: whoops missed that one, thanks!

**Pull request merged**